docs/source/start.ipynb

```diff
@@ -80,6 +80,26 @@
     "X_train, X_holdout, y_train, y_holdout = evalml.preprocessing.split_data(X, y, test_size=.2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To provide data to EvalML, it is recommended that you create a `DataTable` object using [the woodwork project](https://woodwork.alteryx.com/en/stable/).\n",
+    "\n",
+    "EvalML also accepts and works well with pandas `DataFrames`. But using the `DataTable` makes it easy to control how EvalML will treat each feature, as a numeric feature, a categorical feature, a text feature or other type of feature. Woodwork `DataTable`s include features like inferring when a categorical feature should be treated as a text feature. For this reason, if you don't provide woodwork objects, EvalML will raise a warning."
```

Comment:
```suggestion
    "EvalML also accepts and works well with pandas `DataFrames`. But using the `DataTable` makes it easy to control how EvalML will treat each feature, as a numeric feature, a categorical feature, a text feature or other type of feature. Woodwork's DataTable includes features like inferring when a categorical feature should be treated as a natural language feature. For this reason, if you don't provide woodwork objects, EvalML will raise a warning."
```

Reply:
@gsheni ah right. I think we're gonna stick with "text" in evalml for the time being. Some of our classes use "text". We can have a separate conversation about that if you'd like.

I'll add in your other suggestion.

Thanks!